docs/source/customizations/coprocessor.mdx

```diff
@@ -825,6 +825,8 @@ When `stage` is `SupergraphResponse`, if present and `true` then there will be s
 
 An object mapping of all HTTP header names and values for the corresponding request or response.
 
+Headers should be handled like HTTP headers in general and normalized before your coprocessor operates on them. 
```

Comment:
We're telling the reader to do this right? Perhaps clearer to use active case then. (Rather than making it ambiguous whether we/Apollo generally handles headers like HTTP headers)

```suggestion
Ensure headers are handled like HTTP headers in general. For example, normalize header case before your coprocessor operates on them. 
```